drivers/gpio/gpio_nrfx.c

```diff
@@ -132,6 +132,12 @@ static int gpio_nrfx_config(struct device *port, int access_op,
 	u8_t from_pin;
 	u8_t to_pin;
 
+	/* Open Source is not supported on this platform */
```

Comment:
There was a problem with the testcase, I needed to add pull up / pull down on a pin. I couldn't rely on the LED and resistor to set pin value. The test is passing on nrf52840_pca10056 board. The platform supports Open Drain / Open Source mode.